.gitignore

```diff
@@ -19,4 +20,11 @@ otel_rules
 otel_pkgdep
 opentelemetry-go-auto-instrumentation
 *_cache
-example/demo/demo
\ No newline at end of file
+example/demo/demo
+test/zap/test_zap
+test/gorm/v1.22.0/test_gorm_crud
```

Comment:
I believe there are more binaries to ignore

Reply:
Yes, let's add them when we need to

Reply:
The plugin tests should be runned in Go1.22 and Go1.23 now according to the latest otel sdk